shared/store/configure-store.js

```diff
@@ -8,6 +8,11 @@ import {enableStoreLogging, enableActionLogging, closureStoreCheck} from '../loc
 import {actionLogger} from './action-logger'
 import {closureCheck} from './closure-check'
 
+import createSagaMiddleware from 'redux-saga'
+import {call} from 'redux-saga/effects'
+
+import {searchSaga} from '../actions/search'
```

Comment:
does it make sense for us to export 1 default saga from each 'section' ? so more
```
import searchSaga from '../actions/search
```

and if search has something complex it'd have its own forks/ whatever?